components/blitz/src/ome/formats/importer/cli/CommandLineImporter.java

```diff
@@ -492,19 +488,31 @@ public static void advUsage() {
     private static List<Annotation> toTextAnnotations(
                    List<String> namespaces, List<String> strings)
     {
-        if (namespaces.size() != strings.size())
-        {
-            throw new IllegalArgumentException(String.format(
-                            "#Namespaces:%d != #Text:%d", namespaces.size(),
-                            strings.size()));
-        }
         List<Annotation> annotations = new ArrayList<Annotation>();
-        for(int i = 0; i < namespaces.size(); i++)
+        CommentAnnotationI annotation;
+        if (namespaces.size() != strings.size())
```

Comment:
tried this : `./bin/omero import --annotation-text testAnnotation1 /Users/bramalingam/Desktop/Screen\ Shot\ 2017-05-09\ at\ 11.57.33.png`, here namespace.size() != strings.size(), and thus no namespace will be set on this annotation and am getting the following error: 

Exception in thread "main" java.lang.IllegalArgumentException: #Namespaces:0 != #Text:1
	at ome.formats.importer.cli.CommandLineImporter.toTextAnnotations(CommandLineImporter.java:497)
	at ome.formats.importer.cli.CommandLineImporter.main(CommandLineImporter.java:931)